### PR TITLE
AnimesHouse's visual and logo update

### PR DESCRIPTION
**Pull request by DarkstaIkers**
Animeshouse has been update its visual and logo, so I made the changes.

### CODE REVIEW

File: websites/A/Animes House/dist/metadata.json [hunk 1]
```diff
@@ -10,9 +10,9 @@
     "pt": "Um dos sites brasileiros mais confortáveis para assistir animes."
   },
   "url": "animeshouse.net",
-  "version": "1.0.5",
-  "logo": "https://i.imgur.com/o8Px401.png",
-  "thumbnail": "https://i.imgur.com/1YqKDfY.png",
+  "version": "1.0.7",
+  "logo": "https://i.imgur.com/CDlLwID.png",
```
Comment (ririxidev) [inline websites/A/Animes House/dist/metadata.json:14]: it is not 512x512

File: websites/A/Animes House/dist/metadata.json [hunk 2]
```diff
@@ -10,9 +10,9 @@
     "pt": "Um dos sites brasileiros mais confortáveis para assistir animes."
   },
   "url": "animeshouse.net",
-  "version": "1.0.5",
-  "logo": "https://i.imgur.com/o8Px401.png",
-  "thumbnail": "https://i.imgur.com/1YqKDfY.png",
+  "version": "1.0.7",
```
Comment (ririxidev) [inline websites/A/Animes House/dist/metadata.json:13]: 1.0.6

**Review by ririxidev — CHANGES_REQUESTED** [review]
next time talk in english 😉

File: websites/A/Animes House/dist/metadata.json (hunk 1 above)
Comment (DarkstaIkers) [reply websites/A/Animes House/dist/metadata.json:14]: what can I do

File: websites/A/Animes House/dist/metadata.json (hunk 2 above)
Comment (DarkstaIkers) [reply websites/A/Animes House/dist/metadata.json:13]: really?

File: websites/A/Animes House/dist/metadata.json (hunk 1 above)
Comment (ririxidev) [reply websites/A/Animes House/dist/metadata.json:14]: resize it?

Comment (DarkstaIkers) [reply websites/A/Animes House/dist/metadata.json]: alright

**Review by DarkstaIkers — COMMENTED** [review]
okay

File: websites/A/Animes House/dist/metadata.json (hunk 2 above)
Comment (DarkstaIkers) [reply websites/A/Animes House/dist/metadata.json]: okay

**Review by DarkstaIkers — COMMENTED** [review]
Done

**Review by DarkstaIkers — COMMENTED** [review]
That's it?

**Comment by ririxidev** [comment]
logo is still not resized to 512x512

**Review by DarkstaIkers — COMMENTED** [review]
Logo in 512x512 added

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof that you changed the logo in the app too. (App assets)

**Comment by DarkstaIkers** [comment]
> Upload proof that you changed the logo in the app too. (App assets)

I did not understand you question, could you clarify?

**Comment by Bas950** [comment]
> > Upload proof that you changed the logo in the app too. (App assets)
> 
> I did not understand you question, could you clarify?

**Comment by DarkstaIkers** [comment]
> > > Upload proof that you changed the logo in the app too. (App assets)
> > 
> > 
> > I did not understand you question, could you clarify?
> 
> image

Okay, it's done. Do I need send something for you?

**Comment by Bas950** [comment]
> > > > Upload proof that you changed the logo in the app too. (App assets)
> > > 
> > > 
> > > I did not understand you question, could you clarify?
> > 
> > 
> > image
> 
> Okay, it's done. Do I need send something for you?

yea just upload a screenshot of it

**Comment by DarkstaIkers** [comment]
> > > > > Upload proof that you changed the logo in the app too. (App assets)
> > > > 
> > > > 
> > > > I did not understand you question, could you clarify?
> > > 
> > > 
> > > image
> > 
> > 
> > Okay, it's done. Do I need send something for you?
> 
> yea just upload a screenshot of it

6AahoMC
it's done

**Comment by ririxidev** [comment]
okay looks like you are not owner of this presence so you can't update itxd

**Pull request closed without merging**